app/components/recommended-video.js

```diff
@@ -26,8 +27,14 @@ export default Component.extend(NoScrollMixin, {
 				action: trackActions.impression,
 			});
 
-			// Uncomment after XW-4771 test is done
-			// this.initRecommendedVideo();
+			const isClickToPlay = inGroup('RECOMMENDED_VIDEO_AB', 'CLICK_TO_PLAY');
```

Comment:
as we load AbTesting module asynchronously are we sure it will be always available here?

Reply:
Probably yes as we wait 3s but who knows... :P